scout/adapter/mongo/institute.py

```diff
@@ -192,6 +172,24 @@ def institute(self, institute_id):
 
         return institute_obj
 
+    def safe_genes_filter(self, institute_id):
+        """Returns a list of "safe" HGNC IDs to filter variants with. These genes are retrieved from the institute.gene_panels_matching
+        Can be used to limit secondary findings when retrieving other causatives or matching managed variants
+
+        Args:
+            institute_id(str): _id of an institute
+
+        Returns:
+            safe_genes(list of HGNC ids)
+        """
+        safe_genes = []
+        institute_obj = self.institute(institute_id)
+        if not institute_obj:
```

Comment:
...and remove the following line and reindent? Up to you; this may actually be more Pythonesque, but I find the c-style easier here since it avoids a negation and an extra line, plus a little confusion as to the value of `safe_genes == []` for just one indent level extra. 🤷‍♂️

Reply:
I think I'd leave it this was, it's how we handle these situations in most of the code (whenever we touch it). But I could add a comment perhaps?

Reply:
Nah, in that case make it a function call! But its all ok, its just a few words - I can still read it! 😁